### ISSUE TITLE
Add accessibility option to disable animated background on blowfish.page

### ISSUE THREAD

**Issue by HactarCE** [nunocoracao/blowfish#2147]
The Blowfish theme looks lovely and the animated background on blowfish.page is gorgeous but unfortunately it gives me a headache when I am trying to read the documentation, even when it's blurred. This theme is borderline unusable for me simply because I find it so uncomfortable to read the docs for it. (For what it's worth I do have a sensory processing disorder, but I can't think of any other website where I've had a similar issue.)

Is there some way I can make the background static, even if I have to manually modify the HTML every time?

Ideally I would love to see be a site-wide accesibility option to disable the animated background, or for it to only appear on the homepage.

Thank you!

**Comment by BasilSkrnk** [comment]
On top of that, my (Intel) MacBook thinks we’re flying somewhere, and it should help by spinning its fan as loudly as possible.

**Comment by BasilSkrnk** [comment]
Hey, sorry for bumping the old thread. Just wanted to clarify, is it the new option for all the blowfish sites? 

But it’s not enabled for [Blowfish.page](https://blowfish.page/), right? I’m trying to find it there, and I cannot find any a11y options. If not, could we have it, please? Asking because I’m getting a bit dizzy when reading the documentation there, with the background constantly moving.

Thanks!

**Comment by ZhenShuo2021** [comment]
Hi @BasilSkrnk,

The accessibility options are not available on the documentation site because the owner [disabled this feature](https://github.com/nunocoracao/blowfish/commit/4e65fec48fe1432df785562404d453cc54f26c26) in the site configuration.

**Comment by ZhenShuo2021** [comment]
Try the following workarounds:

1. Use a browser extension that blocks JavaScript. This disables the blur, though the background SVG will still move.
2. In Chrome, open DevTools with F12, press ESC to toggle the bottom drawer, and in `Network request blocking`, add `*background*` to block the background image. You can do the similar thing on Firefox too but it's in the network panel not bottom drawer.

Don't know why a11y was disabled on the documentation, so for now these hacky methods are the only way to fix the problem.

**Comment by BasilSkrnk** [comment]
Before it’s enabled (if ever), I made my own local fork, and hosted the documentation on my local server. Feels less hacky, yet it’s cool to know you found your way around this.
